test/e2e/flowaggregator_test.go

```diff
@@ -532,6 +504,102 @@ func testHelper(t *testing.T, data *TestData, podAIPs, podBIPs, podCIPs, podDIPs
+			}
+			t.Logf("command: %s", command)
+			stdout, stderr, err := runAntctl(podName, command, data)
+			if err != nil {
+				t.Fatalf("Error when running `antctl get flowrecords -o json` from %s: %v\n%s", podName, err, antctlOutput(stdout, stderr))
+			}
+
+			var records []map[string]interface{}
+			err = json.Unmarshal([]byte(stdout), &records)
+			if err != nil {
+				t.Fatalf("Error when parsing flow records from antctl")
+			}
+			assert.Equalf(t, 1, len(records), "Number of flow records from antctl is not correct")
+			checkAntctlRecord(t, records[0], srcIP, dstIP, srcPort, dstPort, isIPv6)
+
+		})
+	})
+}
+
+func checkAntctlRecord(t *testing.T, record map[string]interface{}, srcIP, dstIP, srcPort, dstPort string, isIPv6 bool) {
+	assert := assert.New(t)
+	if isIPv6 {
+		assert.Equalf(srcIP, record["sourceIPv6Address"], "The record from antctl does not have correct sourceIPv6Address")
+		assert.Equalf(dstIP, record["destinationIPv6Address"], "The record from antctl does not have correct destinationIPv6Address")
+	} else {
+		assert.Equalf(srcIP, record["sourceIPv4Address"], "The record from antctl does not have correct sourceIPv4Address")
+		assert.Equalf(dstIP, record["destinationIPv4Address"], "The record from antctl does not have correct destinationIPv4Address")
+	}
+	assert.Equalf(srcPort, fmt.Sprintf("%v", record["sourceTransportPort"]), "The record from antctl does not have correct sourceTransportPort")
+	assert.Equalf("perftest-a", record["sourcePodName"], "The record from antctl does not have correct sourcePodName")
+	assert.Equalf("antrea-test", record["sourcePodNamespace"], "The record from antctl does not have correct sourcePodNameSpace")
+	assert.Equalf(controlPlaneNodeName(), record["sourceNodeName"], "The record from antctl does not have correct destinationNodeName")
+
+	assert.Equalf(dstPort, fmt.Sprintf("%v", record["destinationTransportPort"]), "The record from antctl does not have correct destinationTransportPort")
+	assert.Equalf("perftest-b", record["destinationPodName"], "The record from antctl does not have correct destinationPodName")
+	assert.Equalf("antrea-test", record["destinationPodNamespace"], "The record from antctl does not have correct destinationPodNameSpace")
+	assert.Equalf(controlPlaneNodeName(), record["destinationNodeName"], "The record from antctl does not have correct destinationNodeName")
+
+	assert.Equalf(`{"antrea-e2e":"perftest-a","app":"perftool"}`, record["sourcePodLabels"], "The record from antctl does not have correct sourcePodLabels")
+	assert.Equalf(`{"antrea-e2e":"perftest-b","app":"perftool"}`, record["destinationPodLabels"], "The record from antctl does not have correct destinationPodLabels")
+
+	assert.Equalf(fmt.Sprintf("%v", ipfixregistry.FlowTypeIntraNode), fmt.Sprintf("%v", record["flowType"]), "The record from antctl does not have correct flowType")
+	assert.Equalf("6", fmt.Sprintf("%v", record["protocolIdentifier"]), "The record from antctl does not have correct flowType")
```

Comment:
Instead of directly using "6" we can define a constant or reuse some identifier from library to represent tcp protocol.

Reply:
Not found a reusable identifier, defined a constant.